src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -420,7 +420,32 @@ module String =
         elif value.Contains(pattern) then
             Some value
         else None
-     
+    
+    open System.IO
+
+    let getLines (str: string) =
+        use reader = new StringReader(str)
+        [|
+        let line = ref (reader.ReadLine())
```

Comment:
The code was with mutable but had to change it to ref for compiling with VS2013.

The value is mutated in a closure.